tox.ini

```diff
@@ -21,9 +21,11 @@ deps =
     mock
     coverage
     pytest-cov
+    etcd
+    maps
     cover: codecov
 commands =
-    {envpython} -m pytest --cov=tendrl {posargs: tendrl/commons/tests}
+    {envpython} -m pytest --cov=tendrl --cov-report term --cov-report html:../htmlcov {posargs: tendrl/commons/tests}
```

Comment:
Since these changes are for your devel environment, please dont commit them, keep them local